tripper/backends/sparqlwrapper.py

```diff
@@ -39,6 +39,31 @@ def __init__(self, base_iri: str, **kwargs) -> None:
         )  # database is not used in the SPARQLWrapper backend
         self.sparql = SPARQLWrapper(endpoint=base_iri, **kwargs)
 
+    def query(
+        self, query_object, **kwargs
```

Comment:
Silence the pylint error
```suggestion
        self, query_object, **kwargs  # pylint: disable=unused-argument
```

Reply:
It is a good idea to install pre-commit. In your development virtualenv do the following

    pip install -U -e .[dev]
    pre-commit install
    pre-commit run --all  # Force to run all checks

Pre-commit is run together with pytest as CI tests on GitHub, so to avoid failing CI tests it is a good idea to run both before pushing.